packages/common/src/eips/2315.json

```diff
@@ -4,7 +4,7 @@
   "comment": "Simple subroutines for the EVM",
   "url": "https://eips.ethereum.org/EIPS/eip-2315",
   "status": "Draft",
-  "minimumHardfork": "berlin",
+  "minimumHardfork": "istanbul",
```

Comment:
This is a fix, EIP can technically be applied earlier than `berlin`, this would otherwise not allow to instantiate the VM with istanbul and just activate this single EIP.